alfalfa_web/server/dbops.js

```diff
@@ -28,114 +20,105 @@ function currentWinningValue(array) {
   return null;
 }
 
-function setOrNullArray(array, val, level, who) {
-  if (val || val === 0) {
-    array.val[level - 1] = val;
-    array.who[level - 1] = who;
-  } else {
-    array.val[level - 1] = null;
-    array.who[level - 1] = null;
-  }
-}
-
 // Get a point by siteRef and display name
 function getPoint(siteRef, name, db) {
   const mrecs = db.collection("recs");
   return mrecs.findOne({ site_ref: siteRef, "rec.dis": `s:${name}` });
 }
 
-function writePoint(id, siteRef, level, val, who, dur, db) {
+async function getWriteArray(siteRef, id, redis) {
+  const key = `site:${siteRef}:point:${id}`;
```

Comment:
Do we want to move this to a function? It gets used enough that if we ever wanted to change this it would be easier to do it all in the same place.